Frontend/implementations/typescript/src/player.ts

```diff
@@ -17,7 +17,7 @@ document.body.onload = function() {
     Logger.InitLogging(LogLevel.Warning, true);
 
 	// Create a config object
-	const config = new Config({ useUrlParams: true });
+	const config = new Config({ useUrlParams: true, webSocketProtocols: ["soap","xamp"] });
```

Comment:
Is this necessary? I would prefer to keep it exactly as it was (for compatibility reasons) for the reference implementation in case people built around it being a certain way in the past.

Reply:
Okay, let's fix that up and I will approve.